stock_picking_backorder_strategy/models/stock_picking.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _check_backorder(self):
+        # If strategy == 'manual', let the normal process going on
+        self = self.filtered(lambda p: p.picking_type_id.backorder_strategy == "manual")
```

Comment:
Just 1 little note here. I had sometimes a problem with infinite recursion on a similar idiom, not identical. Changing super recordset argument and calling super() although in that case it was to effectively process lines one by one.

I guess tests are passing. So just a note.